### ISSUE TITLE
 Error Running Benchmarks: No Test Files Found

### ISSUE THREAD

**Issue by sarakhosrozadeh** [corazawaf/coraza-benchmark#1]
I am trying to benchmark the Coraza project following the guide provided in the [Coraza benchmark documentation](https://coraza.io/docs/reference/benchmarks/). When I run the benchmark command, I encounter an error stating that there are no test files.

To Reproduce

    Clone the repository: git clone https://github.com/jptosso/coraza-benchmarks
    Navigate to the repository directory: cd coraza-benchmarks
    Run the benchmark command: go test -bench=. ./...

Expected behavior
The benchmarks should run without errors.

Actual behavior
The following error is displayed:

bash

?       github.com/jptosso/coraza-benchmarks    [no test files]

Desktop:

    OS: Kali Linux
    Go version: go1.21.7

I believe the issue is that there are no test files in the repository. Could you please provide guidance on what kind of test files are required to successfully run the benchmarks? Any examples or additional steps would be greatly appreciated.

Thank you for your support and for developing such a valuable tool.

**Comment by jcchavezs** [comment]
This is a very old repository pointing to @jptosso's v2. Unless we port it
to v3 we should archive this cc @fzipri

**Comment by fzipi** [comment]
Yeah, agreed. I think having this is valuable information, but we need to figure it out if we have the bandwidth for updating...
